tests/functional/test_adcm_upgrade.py

```diff
@@ -60,6 +66,13 @@ def _create_host(sdk_client_fs: ADCMClient, bundle_dir: str = "hostprovider") ->
     return provider.host_create(fqdn=f"test_host_{random_string()}")
 
 
+@allure.step("Check actions availability")
+def _assert_available_actions(obj: _BaseObject):
+    obj.reread()
+    actions = [action.name for action in obj.action_list()]
+    assert sorted(actions) == sorted(AVAILABLE_ACTIONS)
```

Comment:
Guess it'll be good to add an assertion message here.
And you may use sets instead of lists.